source/AndroidXProject.cshtml

```diff
@@ -108,6 +109,9 @@
     <TransformFile Include="..\..\source\@(Model.MavenGroupId)\@(Model.Name)\transforms\*.xml">
         <Link>Transforms/%(RecursiveDir)/%(Filename)%(Extension)</Link>
     </TransformFile>
+    <AndroidJavaSource Include="..\..\source\@(Model.MavenGroupId)\@(Model.Name)\java\*.java">
```

Comment:
Are there source files actually being included now?  Curious what these have in them?

Reply:
@redth 
That java source file is a workaround/fix for issue  where `TabLayout.IOnTabSelectedListener2` could not be used due to google's addition of in-between layer in inheritance chain.

Issue (was moved to java.interop repo):

https://github.com/xamarin/java.interop/issues/590#issuecomment-594028778

Java File itself:

https://github.com/xamarin/AndroidX/pull/70/files#diff-cea9a1a8a9a88025a4a69a2b540b0a4aR1-R9 

I could move this part in if block so it would be only for Material project.

Reply:
I must finish this java source and hand bind it.

Reply:
Removed unnecessary methods according to jonp's suggestion. 
Wasn't able to add conditional code in razor template (errors with closing xml tag), so added MSBuild conditon.